Top/argdecode.c

```diff
@@ -342,8 +342,9 @@ static const char *longUsageList[] = {
         "--aft-zero              set aftertouch to zero, not 127 (default)"),
     Str_noop("--limiter[=num]         include clipping in audio output"),
     Str_noop("--vbr                   set MPEG encoding to variable bitrate"),
-    Str_noop("--suppress-version      do not print version details")
-    Str_noop("--run-unit-tests         enable assertion opcodes and report test failures"),
+    Str_noop("--suppress-version      do not print version details"),
+    Str_noop("--print-version         print version and exit"),
```

Comment:
But if --version already prints the version and exits, what does --print-version do differently?